galsim/fits.py

```diff
@@ -537,6 +540,10 @@ def write(image, file_name=None, dir=None, hdu_list=None, clobber=True, compress
     """
     from galsim._pyfits import pyfits
 
+    if image.iscomplex:
```

Comment:
FWIW, I think for that use case, you will typically have real-valued k-space images, so you can just do `kimage.real.write('kim.fits')`.